### PR TITLE
chore: mention v3 in readme

### PR DESCRIPTION
**Pull request by SimenB**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ When you build plugins for Fastify and you want that them to be accessible in th
 1. Use the `skip-override` hidden property
 2. Use this module
 
-__Note: the v2.x series of this module covers Fastify v3. For Fastify v2
+__Note: the v2.x & v3.x series of this module covers Fastify v3. For Fastify v2
 support refert to the v1.x series.__
```
Comment (jsumners) [inline README.md:13]:
```suggestion
 support refer to the v1.x series.__
```

Comment (SimenB) [reply README.md:13]: image

🙁

Comment (SimenB) [reply README.md:13]: One-lined it (seems most paragraphs in this document is one line anyways) and fixed the typo

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**